node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/orchestrator/OrchestratorImpl.java

```diff
@@ -56,10 +55,10 @@ public void suspend(final String hostName) {
     public void suspend(String parentHostName, List<String> hostNames) {
         final BatchOperationResult batchOperationResult;
         try {
-            batchOperationResult = configServerApi.put(
-                    ORCHESTRATOR_PATH_PREFIX_HOST_SUSPENSION_API,
-                    Optional.of(new BatchHostSuspendRequest(parentHostName, hostNames)),
-                    BatchOperationResult.class);
+            String params = String.join("&hostname=", hostNames);
+            String url = String.format("%s/%s?hostname=%s", ORCHESTRATOR_PATH_PREFIX_HOST_SUSPENSION_API,
```

Comment:
hostNames is supposed to contain parentHostName already (if applicable)

Reply:
What? Note that this only changes the REST API, not the data being passed.

Reply:
Nm, I was confused by both ?hostname= and &hostname=